packages/realm/bindgen/src/templates/typescript.ts

```diff
@@ -52,6 +52,7 @@ const PRIMITIVES_MAPPING: Record<string, string> = {
   "bson::BsonDocument": "Record<string, EJson>",
   "bson::BsonArray": "EJson[]",
   QueryArg: "(MixedArg | MixedArg[])",
+  "std::uint_fast16_t": "number",
```

Comment:
I'd suggest trying to declare the proxy port as `uint64_t` in the spec.yml instead.